packages/taquito-contracts-library/test/taquito-contracts-library.spec.ts

```diff
@@ -83,39 +83,35 @@ describe('ContractsLibrary tests', () => {
   it('throw an InvalidAddress error if the contract address is invalid', () => {
     const contractAddress = 'KTinvalid';
     const contractLib = new ContractsLibrary();
-
-    try {
+    expect(() =>
       contractLib.addContract({
         [contractAddress]: {
           script,
           entrypoints,
         },
-      });
-    } catch (e: any) {
-      expect(e).toBeInstanceOf(InvalidAddressError);
-      expect(e.message).toEqual(`Address is invalid: ${contractAddress}`);
-      expect(e).toBeInstanceOf(Error);
-    }
+      })
+    ).toThrow(InvalidAddressError);
+    expect.objectContaining({
```

Comment:
Something is missing here. `expect.objectContaining` cannot be called just like that. We can change the expected string, and the test will still pass.

Reply:
i have fixed the examples where this mistake was made